docs/FAQ.md

```diff
@@ -12,7 +12,7 @@ The Co-dfns compiler is designed to enable you as an APL programmer to use Dyalo
 
 **Will Co-dfns compile my code?**
 
-That depends. Please check out the limitations file for information about what Co-dfns does not support right now. If you have some code that Co-dfns cannot currently compile, that you'd really like it to compile, talk to me and let's see if we can't make that happen. 
```

Comment:
This is a sort of stylistic idiom that I'm playing on, so I'm not sure if I want to remove this from my documentation because of the added flavor.

Reply:
Sorry about that, my English skills didn't understand that was an idiom, I thought it was a typo. Should be reverted in 
102da75 .